parachain/src/chain_spec.rs

```diff
@@ -321,6 +321,7 @@ fn testnet_genesis(
         vault_registry: VaultRegistryConfig {
             minimum_collateral_vault: vec![(CurrencyId::KSM, 0)],
             punishment_delay: DAYS,
+            system_collateral_ceiling: vec![(CurrencyId::DOT, 1_000_000_000_000_000)],
```

Comment:
```suggestion
            system_collateral_ceiling: vec![(CurrencyId::KSM, 1_000_000_000_000_000)],
```

Reply:
Also maybe add a helper function for this and the case below to compute the planck amount.